src/components/common/PageLayout/index.tsx

```diff
@@ -1,43 +1,46 @@
 import { useEffect, useState, type ReactElement } from 'react'
-import cn from 'classnames'
-import { Drawer } from '@mui/material'
 import { useRouter } from 'next/router'
+import cn from 'classnames'
 
 import Sidebar from '@/components/sidebar/Sidebar'
 import Header from '@/components/common//Header'
 import css from './styles.module.css'
 import SafeLoadingError from '../SafeLoadingError'
 import Footer from '../Footer'
 import { AppRoutes } from '@/config/routes'
+import { useSafeAppUrl } from '@/hooks/safe-apps/useSafeAppUrl'
+import { useMediaQuery } from '@mui/material'
 
 const PageLayout = ({ children }: { children: ReactElement }): ReactElement => {
   const router = useRouter()
-  const [isMobileDrawerOpen, setIsMobileDrawerOpen] = useState<boolean>(false)
-  const hideSidebar = router.pathname === AppRoutes.share.safeApp
+  const isMobile = useMediaQuery('(max-width: 900px)')
```

Comment:
We try to avoid using JS for responsiveness. Why is this needed?

Reply:
I think that we need this because we need to set the variant of the Drawer based on the resolution in the Sidebar component:

```
      <Drawer
        variant={isMobile ? 'temporary' : 'persistent'}
```

Reply:
I mean, before we were rendering two sidebars, for mobile and for desktop, why did you have to change that?